airflow/www/static/js/connection_form.js

```diff
@@ -87,7 +34,7 @@ $(document).ready(function () {
       $(this).text($(this).attr("orig_text"));
     });
     $(".form-control").each(function(){$(this).attr('placeholder', '')});
-
+    let config = JSON.parse($("#field_parameters").text())
```

Comment:
Strange. It worked.

Reply:
Right :facepalm: `compile_assets`

Reply:
We really need to fail webserver when assets are not "rebuilt"

Reply:
It doesn't help with the "rebuild on deman" but if you run `yarn --cwd airflow/www dev` it will run a compiler that watches for changes and automatically rebuilds assets. I _think_ it also doesn't fingerprint the assets, so the manifest at the start is valid even when content changes.

Reply:
Oh, no I'm wrong. When running in dev mode it still fingerprints, so the file name changes when the input does, meaning that the webserver needs to be restarted.